mule-packager/src/main/java/org/mule/tools/api/verifier/MuleProjectVerifier.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Mule ESB Maven Tools
+ * <p>
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * <p>
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.tools.api.verifier;
+
+import org.mule.tools.api.exception.ValidationException;
+
+public class MuleProjectVerifier implements ProjectVerifier {
+
+  @Override
+  public void verify() throws ValidationException {
```

Comment:
Why is this empty?

Reply:
right now we only do verifications for the mule policy, I left  a stub for any future verifications.